packages/cli/src/bin/pipcook-plugin.ts

```diff
@@ -69,4 +113,15 @@ program
   .option('-d|--datatype <name>', 'the plugin datatype')
   .action(list);
 
+program
+  .command('upload <localPluginsPath>')
```

Comment:
Why do we add an extra command? I think we should keep it consistent with `pipcook plugin install`.

Reply:
The command `plugin install` supported local path, we can't distinguish between them.

Reply:
Why not replacing the local path with upload?